.vscode/import_map.json

```diff
@@ -3,6 +3,7 @@
     "THIS FILE EXISTS ONLY FOR VSCODE! IT IS NOT USED AT RUNTIME": {}
   },
   "imports": {
+    "@vscode_787_hack/": "../tests/fixture_tailwind_remote_classes/",
```

Comment:
Is this here deliberately? This import is already defined in `tests/fixture_tailwind_remote_classes/deno.json`.

Reply:
It is, in order to get vsocde to behave. The issue is https://github.com/denoland/vscode_deno/issues/787. Note how the path here is slightly different.